src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -960,6 +968,36 @@ export class NetworkAreaDiagramViewer {
         });
     }
 
+    moveEdgeName(
+        edgeNode: SVGGraphicsElement,
+        anchorPoint: Point,
+        edgeStart: Point
+    ) {
+        const edgeNameElements = DiagramUtils.getEdgeNameElements(edgeNode);
```

Comment:
as the content of `DiagramUtils.getEdgeNameElements` is very compact with querySelectors, I think you can remove it and call the querySelectors directly here

Reply:
Removed DiagramUtils function and used querySelector for getting the elements in this method